webpack.config.ts

```diff
@@ -107,6 +110,9 @@ module.exports = {
       commonPrefix + '/pages/topic-viewer-page/topic-viewer-page.scripts.ts',
   },
   plugins: [
+    new CKEditorWebpackPlugin({
+      language: 'pl'
```

Comment:
What's this?

Reply:
Sorry, default config of CK-5 Updated to 'en' to avoid confusion.
this is used for i18n-translations as well